docs/zen.css

```diff
@@ -4,7 +4,6 @@
 /*! Zen Cart Overrides */
 body {
   background: #fff;
-  color: #777;
```

Comment:
I agree that the 777 grey is weak contrast.

And yet flipping the grey all the way to black negatively affects some of the aesthetics.

How about a slighter darkening by changing `#777` to `#606060`?

Reply:
#303030 looks better.  Old people need strong contrast.